internal/uvm/create_lcow.go

```diff
@@ -99,7 +99,11 @@ func NewDefaultOptionsLCOW(id, owner string) *OptionsLCOW {
 		opts.Owner = filepath.Base(os.Args[0])
 	}
 
-	// TODO: If > RS5 use KernelDirect. If rootfs.vhd exists use it.
+	if _, err := os.Stat(filepath.Join(opts.BootFilesPath, "rootfs.vhd")); err != nil {
+		// We have a rootfs.vhd in the boot files path. Use it over an initrd.img
+		opts.RootFSFile = "rootfs.vhd"
```

Comment:
Should we put roofs.vhd as a constant rather than a string literal?

Reply:
Yea I'll add them for this and `initrd.img`